app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -67,6 +67,22 @@ def import_package_profile(simple_packages)
           self.installed_packages.create!(:name => simple_package.name, :nvra => simple_package.nvra) unless existing_nvras.include?(simple_package.nvra)
         end
       end
+
+      def subscription_status(options = {})
+        @subscription_status ||= get_status(::Katello::SubscriptionStatus).to_status(options)
+      end
+
+      def subscription_status_label(options = {})
+        @subscription_status_label ||= get_status(::Katello::SubscriptionStatus).to_label(options)
+      end
+
+      def errata_status(options = {})
+        @errata_status ||= get_status(::Katello::ErrataStatus).to_status(options)
+      end
+
+      def errata_status_label(options = {})
+        @errata_status_label ||= get_status(::Katello::ErrataStatus).to_label(options)
+      end
```

Comment:
@ehelms so actually i didn't realize that foreman was already returning the statuses and labels if these methods exist:

```
"global_status": 0,
"global_status_label": "Warning",
"organization_id": 1,
"organization_name": "Default Organization",
"location_id": 2,
"location_name": "Default Location",
"puppet_status": 0,
"errata_status": 1,
"errata_status_label": "Could not calculate errata status, ensure host is registered and katello-agent is installed",
"subscription_status": 0,
"subscription_status_label": "Fully entitled",
```

So i agree that this isn't ideal, but its what foreman is doing right now.  I could remove these methods and redefine it our own way, or go with this.  Thoughts?